tests/robustness/failpoint/cluster.go

```diff
@@ -304,3 +308,28 @@ func patchArgs(args []string, flag, newValue string) error {
 	}
 	return fmt.Errorf("--%s flag not found", flag)
 }
+
+type growMember struct {
+	currentSize int
```

Comment:
Which "current" you mean? Maybe `initialSize`.

Reply:
I no longer need this initialSize.  It's now `scalingTarget`.